### PR TITLE
Patient banner

### PR DESCRIPTION
**Pull request by barabo**
I've refactored some of the patient context information into a new component, which I've included in both the PAMA and RxView tabs.

### CODE REVIEW

File: tests/components/RxView/rx-view.test.js [hunk 1]
```diff
@@ -117,12 +117,6 @@ describe('RxView component', () => {
     jest.resetModules();
   });
 
-  it('contains relevant patient information if patient is in context', () => {
-    setup(patient, medListPhase, medications, prescription);
-    expect(renderedComponent.find('.patient-data-text').text()).toContain(patient.name);
-    expect(renderedComponent.find('.patient-data-text').text()).toContain(patient.id);
-  });
-
   it('has default view elements', () => {
```
Comment (dennispatterson) [inline tests/components/RxView/rx-view.test.js:120]: With the above test removed, can you update this test to assert that the patient banner is included?

File: src/components/PatientBanner/patient-banner.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styles from './patient-banner.css';
+
+
+function PatientBanner(props) {
```
Comment (dennispatterson) [inline src/components/PatientBanner/patient-banner.jsx]: Can you add some initial tests for this new component (e.g. rendering it with and without the attributes being present)?

**Review by dennispatterson — CHANGES_REQUESTED** [review]

File: tests/components/RxView/rx-view.test.js (hunk 1 above)
Comment (barabo) [reply tests/components/RxView/rx-view.test.js:120]: No problem.

File: src/components/PatientBanner/patient-banner.jsx (hunk 2 above)
Comment (barabo) [reply src/components/PatientBanner/patient-banner.jsx]: Done.  PTAL

**Review by jmandel — APPROVED** [review]
LGTM, thanks!

**Review by dennispatterson — APPROVED** [review]

**Pull request merged**